regression-test/suites/tpcds_sf1_p1/spill_test/q23.groovy

```diff
@@ -0,0 +1,202 @@
+suite("q23") {
+    sql """set enable_force_spill=true"""
```

Comment:
In this test case, we should set minRevocableMem explictly, to make sure spill disk is triggered. Not depend on fuzzy.
Then we could make sure, all PR will trigger for this test case.